test/sai_test/sai_tunnel_test.py

```diff
@@ -180,3 +180,118 @@ def runTest(self):
         finally:
             pass
 
+class IPInIPTunnelEncapv4Inv4Test(IpInIpTnnnelBase):
+    """
+    We will send ipinip packet from lag1 and verify getting inner packet by matching tunnel term table entry, recieving the inner packet on port1 by  matching route entry
+    """
+
+    def setUp(self):
+        """
+        Test the basic setup process.
+        """
+        IpInIpTnnnelBase.setUp(self)
+        
+    def ipv4inipv4ecap(self):
+        """
+        Generate ingress ipinip packet as decribed by Testing Data Packet
+        Send encap packet from lag1.
+        Generate expected decap packet as decribed by Testing Data Packet.
+        Recieve decap packet from port1, compare it with expected decap packet.
```

Comment:
Recieve decap packet from port1, compare it with expected decap packet.?
use the test case steps and title from test plan